.circleci/generate_circleci_config.py

```diff
@@ -11,16 +11,27 @@
 from jinja2 import Environment, FileSystemLoader
 
 IMAGE_MAP = collections.OrderedDict([
-    ("1.10.5-7", ["alpine3.10", "buster", "rhel7"]),
+    ("1.10.5-8.dev", ["alpine3.10", "buster", "rhel7"]),
     ("1.10.6-3", ["alpine3.10", "buster"]),
     ("1.10.7-11", ["alpine3.10", "buster"]),
     ("1.10.10-1", ["alpine3.10", "buster"]),
 ])
 
+# Airflow Versions for which we don't publish Python Wheels
+DEV_WHITELIST = ["1.10.5", "1.10.6"]
```

Comment:
For the versions listed here:

- We won't create Nightly builds
- We won't look for the latest versions in pip.astronomer.io
- We would look for the GIT TAG matching the AC Version